src/server/pfs/cmds/cmds.go

```diff
@@ -647,6 +650,13 @@ $ pachctl put-file repo branch -i file
 # NOTE this URL can reference local files, so it could cause you to put sensitive
 # files into your Pachyderm cluster.
 $ pachctl put-file repo branch -i http://host/path
+
+# Put a pgdump file
+$ pachctl put-file repo branch tablename -f tablename.pgdump --split sql
+
+# Overwrite header/footer for output generated by put-file split
+# To remove the header or footer, specify an empty file
+$ echo '' | pachctl put-file repo branch directory_containing_splitfiles --header=header.txt --footer=footer.txt - 
```

Comment:
This is a slightly awkward case that we may want to think about expanding a bit. Having to pipe in `echo ''` definitely isn't super pretty. You can achieve a similar effect by passing `-f /dev/null`, not sure if that's much better. Maybe we should consider adding `put-header` and `put-footer` which just update the header and footer, and those won't have any weird interactions with the existing `put-file` flags. They could just be different pachctl commands and still send `put-file` requests under the hood.

Reply:
Also this example seems to be invalid because it doesn't have a `--split` flag.

Reply:
Sounds good. Made separate `put-header` and `put-footer` commands